docs/README.adoc

```diff
@@ -0,0 +1,99 @@
+= ShiftStack CI
+
+Each OpenShift on OpenStack continuous integration test job runs on one of two clouds:
+
+* *MOC*: massopen.cloud running OSP 13
+* *Vexxhost*: A private cloud offering likely running OpenStack Ussuri
+
+Currently, we have three different test suites:
+
+* *Parallel*: The Conformance test suite, executed parallelizing tests
+* *Serial*: The Conformance test suite, executed one job at a time
+* *Early*: Smoke tests
+
+Currently, our jobs deploy on clouds with equivalent characteristics:
+
+* Control Plane (and bootstrap) nodes boot from a 25GB high-performance Cinder volume
+* Control Plane: 3 VMs with:
+** 8 vCPUs
+** 16 GB RAM
+** 25 GB disk
+* Compute Nodes: 3 VMs with:
+** 8 vCPUs
+** 16 GB RAM
+** 25 GB disk
+* OpenShift SDN
+* Installer-provisioned network
+* FIP-powered connectivity (we use AWS Route53 as the external DNS)
+* The OpenStack cloud is using a valid HTTPS certificate (no cacert in clouds.yaml)
+* Swift as a backend for cluster-image-registry-operator
+
+.ShiftStack periodics
+|===
+|Installer |Test suite |Job name |Cloud (see top)
```

Comment:
The table would be easier to read if we also add the branch being tested: 4.6, 4.5, ...